.bin/set-up-globals.sh

```diff
@@ -1,12 +1,30 @@
 #!/bin/bash
 set -ex
 
-
 TERMINUS_PLUGINS_DIR=.. terminus list -n remote
 
 set +ex
```

Comment:
once again, unless there's good reason for it, in favor of at least set -e for the whole script and skipping -x.